openpype/hosts/maya/plugins/create/create_render.py

```diff
@@ -80,31 +80,58 @@ def __init__(self, *args, **kwargs):
         if self._project_settings["maya"]["RenderSettings"]["apply_render_settings"]: # noqa
             lib_rendersettings.RenderSettings().set_default_renderer_settings()
 
-        # Deadline-only
+        # Handling farms
         manager = ModulesManager()
         deadline_settings = get_system_settings()["modules"]["deadline"]
-        if not deadline_settings["enabled"]:
-            self.deadline_servers = {}
+        rr_settings = get_system_settings()["modules"]["royalrender"]
+
+        self.deadline_servers = {}
+        self.rr_paths = {}
+
+        if deadline_settings["enabled"]:
+            self.deadline_module = manager.modules_by_name["deadline"]
+            try:
+                default_servers = deadline_settings["deadline_urls"]
+                project_servers = (
+                    self._project_settings["deadline"]["deadline_servers"]
+                )
+                self.deadline_servers = {
+                    k: default_servers[k]
+                    for k in project_servers
+                    if k in default_servers
+                }
+
+                if not self.deadline_servers:
+                    self.deadline_servers = default_servers
+
+            except AttributeError:
+                # Handle situation were we had only one url for deadline.
+                # get default deadline webservice url from deadline module
+                self.deadline_servers = self.deadline_module.deadline_urls
+
+        # RoyalRender only
+        if not rr_settings["enabled"]:
             return
-        self.deadline_module = manager.modules_by_name["deadline"]
+
+        self.rr_module = manager.modules_by_name["royalrender"]
         try:
-            default_servers = deadline_settings["deadline_urls"]
-            project_servers = (
-                self._project_settings["deadline"]["deadline_servers"]
+            default_paths = rr_settings["rr_paths"]
+            project_paths = (
+                self._project_settings["royalrender"]["rr_paths"]
             )
-            self.deadline_servers = {
-                k: default_servers[k]
-                for k in project_servers
-                if k in default_servers
+            self.rr_paths = {
+                k: default_paths[k]
+                for k in project_paths
+                if k in default_paths
             }
 
-            if not self.deadline_servers:
-                self.deadline_servers = default_servers
+            if not self.rr_paths:
+                self.rr_paths = default_paths
 
         except AttributeError:
-            # Handle situation were we had only one url for deadline.
-            # get default deadline webservice url from deadline module
-            self.deadline_servers = self.deadline_module.deadline_urls
+            # Handle situation were we had only one path for royalrender.
+            # Get default royalrender root path from the rr module.
+            self.rr_paths = self.rr_module.rr_paths
```

Comment:
I'd prefer this readability wise more like the if statement for deadline:

```python
if royal_render["enabled"]:
    init_royal_render()
```

Just so it's clearer that what's currently the remainder of that method applies to Royal Render only.